package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "craze",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"start": "pingy dev",
+		"export": "pingy export"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
+		"animate.css": "^3.5.2",
+		"bootstrap": "^4.0.0-beta.2",
```

Comment:
Hello Victor, i will answer some question what you are mentioned.
I used beta bootstrap 4 because i wanted to use the last version of bootstrap and he is in the version 4 beta i wanted to use the new features in this test, but for production I think the most correct would be stable version 3 for now.